artiq/coredevice/coredevice_generic.schema.json

```diff
@@ -98,7 +120,7 @@
             }
         }
     },
-    "required": ["target", "variant", "hw_rev", "base", "peripherals"],
+    "required": ["target", "variant", "hw_rev", "base", "peripherals", "moninj"],
```

Comment:
Is there a need for `moninj` entry to be required? Like does it make sense to use localhost for default address and the default ports stated in the manual/`aqctl_proxy_moninj`?

So far it will break all the existing json configurations, though adding an `moninj` entry should be doable.

Reply:
either we set moninj entries with default values, or the dashboard won't work anymore. iirc it is decided to use moninj proxy for all dashboard interaction so this is pretty much necessary right now